workers/dotnet/Temporalio.Omes.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <OutputType>Exe</OutputType>
+        <OutputType>Library</OutputType>
```

Comment:
I made the lint step compile the project to check it type checks etc and had to change this since there's no `Main`. I see that the build step is generating it's own csproj file, so this seems fine to me.